packages/fxa-auth-server/lib/senders/emails/bin/mjml-sever.ts

```diff
@@ -0,0 +1,56 @@
+import http from 'http';
+import stream from 'stream';
+import net from 'net';
+import { render } from '../renderer';
+
+const { PORT = 8192, HOST = '0.0.0.0' } = process.env;
+
+const MJML_OPTIONS = {};
+
+console.log(`Starting MJML API server at ${HOST}:${PORT}`);
+
+const server = http.createServer(handleRequest);
+server.on('clientError', handleError);
+server.listen({ port: PORT, host: HOST });
+
+async function handleRequest(
+  req: http.IncomingMessage,
+  res: http.ServerResponse
+) {
+  try {
+    const body = await readStream(req);
+    const data = JSON.parse(body.trim());
+    const { template: templateName, ...variables } = data;
+
+    console.log(`REQUEST ${templateName}`);
+
+    const result = render(templateName, variables);
```

Comment:
We should be able to use this render function when building out the feature flagging parts 👍🏽